### PR TITLE
Initital treasury changes

### PR DESCRIPTION
**Pull request by RyanLucchese**
Fixes Issue #95

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -99,11 +99,12 @@ class CMainParams : public CChainParams {
         consensus.nInstantSendKeepLock = 24;
 
         consensus.nBudgetPaymentsStartBlock = 0; // actual historical value
-        consensus.nBudgetPaymentsCycleBlocks = 16616; // ~(60*24*30)/2.6, actual number of blocks per month is 200700 / 12 = 16725
+        consensus.nBudgetPaymentsCycleBlocks = 43200; // ~(60*24*30)/1
         consensus.nBudgetPaymentsWindowBlocks = 100;
         consensus.nBudgetProposalEstablishingTime = 60*60*24; // 1 day
         consensus.nSuperblockStartBlock = 0; //
-        consensus.nSuperblockCycle = 16616; // ~(60*24*30)/2.6, actual number of blocks per month is 200700 / 12 = 16725
+        consensus.nSuperblockFirstBlock = 11520; // After 8 days (60*24*8)
```
Comment (ediston) [inline src/chainparams.cpp:106]: 8 days from ? or you meant every 8 days ?

File: src/chainparams.cpp [hunk 2]
```diff
@@ -99,11 +99,12 @@ class CMainParams : public CChainParams {
         consensus.nInstantSendKeepLock = 24;
 
         consensus.nBudgetPaymentsStartBlock = 0; // actual historical value
-        consensus.nBudgetPaymentsCycleBlocks = 16616; // ~(60*24*30)/2.6, actual number of blocks per month is 200700 / 12 = 16725
+        consensus.nBudgetPaymentsCycleBlocks = 43200; // ~(60*24*30)/1
```
Comment (ediston) [inline src/chainparams.cpp:102]: [EDIT] Can we please document what  is 1.. there was a comment there which missing now :)

**Comment by RyanLucchese** [comment]
test this please

**Comment by RyanLucchese** [comment]
@energicryptocurrency-jenkins test this please

**Comment by RyanLucchese** [comment]
@energicryptocurrency-jenkins test this please

**Comment by RyanLucchese** [comment]
@energicryptocurrency-jenkins test this please

**Comment by RyanLucchese** [comment]
@energicryptocurrency-jenkins test this please

**Comment by RyanLucchese** [comment]
Superseded by PR #178

**Pull request closed without merging**